.gitignore

```diff
@@ -2,4 +2,7 @@ __pycache__/
 .env
 .vscode
 .mypy_cache
-.ipynb_checkpoints/
\ No newline at end of file
+.ipynb_checkpoints/
+.idea/
```

Comment:
These are files which are added by my local environment by IntelliJ IDEA and Python virtualenv, which aren't necessarily in my direct control, so I add them to .gitignore to prevent them from being tracked by git. 

I could also update my global .gitignore if it's an issue, but it's pretty common to see paths like .idea/ in a project .gitignore.

Reply:
+1 to adding this, it prevents many silly mistakes down the line and it doesn't hurt at all to have a couple more lines in this file